lte/gateway/python/integ_tests/Makefile

```diff
@@ -32,13 +32,16 @@ endif
 # any ending statement the failing testcases are missing status update
 define execute_test
 	echo "Running test: $(1)"
+	echo "pass" > $(MAGMA_ROOT)/test_status.txt
 	timeout --foreground -k 930s 900s sudo -E PATH=$(PATH) PYTHONPATH=$(PYTHONPATH):$(S1AP_TESTER_PYTHON_PATH) $(BIN)/pytest $(FLAKY_CMD_ARGS) --capture=tee-sys --junit-xml=$(RESULTS_DIR)/$(basename $(notdir $(1))).xml -x $(1) || (echo "fail" > $(MAGMA_ROOT)/test_status.txt && exit 1)
+	if [ ! -z `grep -s pass $(MAGMA_ROOT)/test_status.txt` ]; then echo "Final status $(1): Passed"; else echo "Final status $(1): Failed"; exit 1; fi
```

Comment:
Do we reach this line with a failing test? If we exit 1 in line 36, the echo command should not be executed.